api/src/wfl/executor.clj

```diff
@@ -332,7 +332,10 @@
   [tx {:keys [details] :as executor}]
   (when-not (postgres/table-exists? tx details)
     (throw (ex-info "Missing executor details table" {:table details})))
-  (let [query "SELECT * FROM %s WHERE workflow IS NOT NULL ORDER BY id ASC"]
+  (let [query "SELECT * FROM %s
+               WHERE workflow IS NOT NULL
+               AND   retry    IS NULL
```

Comment:
Can you explain why we want to exclude records with a populated `retry` from these fetches?

Reply:
Added a long comment